3737/1/parser.rb

```diff
@@ -0,0 +1,19 @@
+require 'nokogiri'
+require 'open-uri'
+require 'xpath'
+
+# class for getting github page
+class Parser
+  def search_repo(gem_name)
+    url = Nokogiri::HTML(open("http://rubygems.org/gems/#{gem_name}"))
+    check_page('code', url) || check_page('home', url)
```

Comment:
I'd move this to method bellow. I'd like to see here something like `gem_github_url(rubygems_page)`